hummingbot/strategy/arbitrage/arbitrage.pyx

```diff
@@ -628,12 +634,13 @@ cdef list c_find_profitable_arbitrage_orders(object min_profitability,
     """
     cdef:
         object step_amount = s_decimal_0
+        _bid_price_adjusted
+        object current_ask_
```

Comment:
Is this by accident?
Apart from comments, we shouldn't make changes to files not required/tested for this feature.